ionoscloud/resource_certificate_manager_auto_certificate_test.go

```diff
@@ -0,0 +1,209 @@
+package ionoscloud
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	certSDK "github.com/ionos-cloud/sdk-go-cert-manager"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/services"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/utils/constant"
+)
+
+func TestAccCertificateManagerAutoCertificate(t *testing.T) {
+	var autoCertificate certSDK.AutoCertificateRead
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCMAutoCertificateDestroyCheck,
+		Steps: []resource.TestStep{
+			{
+				Config: CMAutoCertificateConfig,
+				Check: resource.ComposeTestCheckFunc(
+					CMAutoCertificateExistenceCheck(constant.AutoCertificateResource+"."+constant.TestCMAutoCertificateName, &autoCertificate),
+					resource.TestCheckResourceAttr(constant.AutoCertificateResource+"."+constant.TestCMAutoCertificateName, CMProviderLocationAttr, CMProviderLocationVal),
+					resource.TestCheckResourceAttr(constant.AutoCertificateResource+"."+constant.TestCMAutoCertificateName, CMAutoCertCommonNameAttr, CMAutoCertCommonNameVal),
+					resource.TestCheckResourceAttr(constant.AutoCertificateResource+"."+constant.TestCMAutoCertificateName, CMAutoCertKeyAlgorithmAttr, CMAutoCertKeyAlgorithmVal),
+					resource.TestCheckResourceAttr(constant.AutoCertificateResource+"."+constant.TestCMAutoCertificateName, CMAutoCertNameAttr, CMAutoCertNameVal),
+					resource.TestCheckResourceAttrSet(constant.AutoCertificateResource+"."+constant.TestCMAutoCertificateName, "provider_id"),
+					resource.TestCheckResourceAttrSet(constant.AutoCertificateResource+"."+constant.TestCMAutoCertificateName, "last_issued_certificate_id"),
+					// TODO -- Check that subject_alternative_names contains the proper list
```

Comment:
is this TODO also meant to be merged to master?

Reply:
@avirtopeanu-ionos no, thanks for catching this. I added this `TODO` but forgot to complete it. It is now completed in 151f8a269d16c58497218a43943de98ba9eb83f9.